tests/kafkatest/tests/core/network_degrade_test.py

```diff
@@ -129,10 +129,10 @@ def test_rate(self, task_name, device_name, latency_ms, rate_limit_kbit):
         self.logger.info("Measured rates: %s" % measured_rates)
 
         # We expect to see measured rates within an order of magnitude of our target rate
-        low_kbps = rate_limit_kbit / 10
+        low_kbps = rate_limit_kbit // 10
         high_kbps = rate_limit_kbit * 10
         acceptable_rates = [r for r in measured_rates if low_kbps < r < high_kbps]
 
         msg = "Expected most of the measured rates to be within an order of magnitude of target %d." % rate_limit_kbit
-        msg += " This means `tc` did not limit the bandwidth as expected."
+        msg += " This means `tc` did not limit the bandwidth as expected. Measured rates %s" % str(measured_rates)
```

Comment:
nit: I believe % is a bit deprecated in favour of `.format(..)` or `f"This means .. {measured_rates}"` (for >=3.6).

Reply:
Actually, this change unrelated. Reverted.
I just trying to debug this test, because it fails(it fails in the trunk, also).

Anyway, I think you are right and we can rewrite all usages of `"..." % param` to the new syntax.

Let's do it in another PR?